Rotativa/Rotativa.nuspec

```diff
@@ -2,7 +2,7 @@
 <package >
   <metadata>
     <id>Rotativa</id>
-    <version>1.7.3</version>
+    <version>1.8.0</version>

```

Comment:
i've decided it is _Minor_, cuz I've added backwards compatible functionality to the public API (**ConvertTimeout** which is not required)
See https://semver.org/spec/v2.0.0.html#spec-item-7